tests/flytekit/unit/remote/test_remote.py

```diff
@@ -763,21 +764,22 @@ def my_wf(a: int) -> typing.List[str]:
     with pytest.raises(FlyteAssertion):
         _get_pickled_target_dict(my_wf)
 
+
 def test_get_pickled_target_dict_with_eager():
     @task
     def t1(a: int) -> int:
         return a + 1
 
     @task
-    def t2(a: int) -> int:
+    async def t2(a: int) -> int:
         return a * 2
 
     @eager
     async def eager_wf(a: int) -> int:
-        out = await t1(a=a)
+        out = t1(a=a)
         if out < 0:
             return -1
-        return await t2(a=out)
+        return t2(a=out)
```

Comment:
what does this case mean? an implicit await at the end of eager_wf exec?

Reply:
reverting, doesn't matter since pickling is not yet enabled for eager tasks, will enable after this initial pr is in.